src/components/CookieBanner/CookieBanner.jsx

```diff
@@ -1,6 +1,6 @@
-import styles from "./CookieBanner.module.scss";
+import styles from './CookieBanner.module.scss'
```

Comment:
I think you should keep the semicolon at the end, as it allows for fewer mistakes in JavaScript :) But it's a style choice, so whichever works just fine technically!